css/style.css

```diff
@@ -56,6 +56,90 @@ a.anchor{
 	}
 }
 
+/*Down arrow animation*/
+.arrows {
+	width: 60px;
+	height: 72px;
+	left: 50%;
+	position: relative;
+	margin-left: -30px;
+	margin-right: auto;
+	margin-top: 50px;
+	bottom: 20px;
+}
+
+.arrows path {
+	stroke: #fff;
+	fill: transparent;
+	stroke-width: 1px;	
+	animation: arrow 2s infinite;
+	-webkit-animation: arrow 2s infinite; 
+}
+
+.arrows-gdpr {
+	width: 60px;
+	height: 72px;
+	position:relative;
+	left: 50%;
+	margin-left: -30px;
+	margin-top: 10px;
+	bottom: 20px;
+}
+
+.arrows-gdpr path {
+	stroke: #7bb69d;
+	fill: transparent;
+	stroke-width: 1px;	
+	animation: arrow 2s infinite;
+	-webkit-animation: arrow 2s infinite; 
+}

```

Comment:
It seems that the only difference between .arrows and .arrows-gdpr are margins. 
I would suggest to still used the class .arrows for the GDPR page and add a class that would modify the margins only for the GDPR page.